index.js

```diff
@@ -99,6 +100,15 @@ module.exports = function (text, opts) {
 
 	var NL = '\n';
 	var PAD = ' ';
+
+	if (opts.align) {
+		text = ansiAlign(text, {
+			align: opts.align,
+			split: NL,
+			pad: PAD
```

Comment:
Changing defaults would be a breaking change and require a major release anyways though.